.gitmodules

```diff
@@ -7,3 +7,4 @@
 [submodule "oasis3-mct"]
 	path = oasis3-mct
 	url = https://github.com/ACCESS-NRI/oasis3-mct.git
+	branch = improve-build-system
```

Comment:
I suppose the oasis3-mct PR should be merged first, then the submodule updated to point to the master branch before merging?

Now I can see how much extra work sub-modules are: making a PR for these changes back to COSIMA would require a similar two-step process.

Reply:
Yes, oasis3-mct PR will need to be merged first. Then this branch will need to point to oasis3-mct master. Merging back to a parent could be done via master to avoid a two-step process.